external/java/src/main/java/ch/epfl/dedis/calypso/Document.java

```diff
@@ -50,14 +49,12 @@ public Document(byte[] data, byte[] keyMaterial, byte[] extraData, DarcId publis
      * @param data        any data that will be stored encrypted on the skipchain.
      *                    There is a 10MB-limit on how much data can be stored. If you
```

Comment:
The limit is based on the block size, which defaults to 4 megs (see byzcoin/service.go, defaultMaxBlockSize). If a given ledger has a higher block size, then more data could be stored in a Document.

It would be helpful to actually test this, because there is likely some overhead meaning that the real max size is 4mb - X, where X = ??.